RomHack.toml

```diff
@@ -1,33 +0,0 @@
-[info]
```

Comment:
I have no idea why git is acknowledging this file.  I had to remove it from my git cache so git would stop telling me its modification date changed.  Same goes for regional-patch.asm